config-cli/src/main/java/com/quorum/tessera/io/IOCallback.java

```diff
@@ -1,30 +0,0 @@
-
-package com.quorum.tessera.io;
-
-import java.io.IOException;
-import java.io.UncheckedIOException;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-/**
- * Callback and template function t uncheck IO exceptions
```

Comment:
typo

Reply:
yes.. it appears to be a typo in a source file that no longer exists and this change enabled other changes I have backed up over three days.